pom.xml

```diff
@@ -17,6 +17,19 @@
 		<version.org.apache.maven.plugins.maven.compiler.plugin>3.8.0</version.org.apache.maven.plugins.maven.compiler.plugin>
 	</properties>
 
+	<dependencyManagement>
```

Comment:
@joaofreitas-qubit I don't really see the need to add the DM here since there are no dependencies in this pom that use this DM.